tests/framework/clients/cluster.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clients
+
+import (
+	"fmt"
+
+	"github.com/rook/rook/pkg/model"
+)
+
+// IsClusterHealthy determines if the Rook cluster is currently healthy or not.
+func IsClusterHealthy(testClient *TestClient) (model.StatusDetails, error) {
+
+	status, err := testClient.Status()
+	if err != nil {
+		return status, err
+	}
+
+	// verify all mons are in quorum
+	if len(status.Monitors) < 1 {
```

Comment:
should we check for 3 mons?

Reply:
sure we can make sure there's at least 3 mons.